src/Controls/tests/UITests/Tests/Issues/Issue19592.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue19592 : _IssuesUITest
+	{
+		public override string Issue => "Span LineHeight Wrong on Android";
+
+		public Issue19592(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public void SpanLineHeightShouldNotGrowProgressively()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
+			_ = App.WaitForElement("label");
+
+			// The line height should be the same for each line
+			// of the paragraph, 1.5 and 2.5 respectively,
+			// as opposed to progressively growing
+			App.Screenshot();
```

Comment:
Could we use here `App.VerifyScreenshot`? This will generate a snapshot to compare it with a reference one.

Reply:
Done